cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,104 @@
++
+VPCs using deprecated AZs aren't eligible for the upgrade.
+
+Legacy VPNs::
++
+VPCs with legacy VPNs configured aren't eligible for the upgrade.
+
+Direct Connect or VPC Peering:: 
+If your VPC has Direct Connect or VPC Peering configured, migrate to Transit Gateway or VPN before upgrading.
+
+TLS 1.1 ciphers::
++
+These ciphers can't be carried over from CloudHub to CloudHub 2.0:
++
+* ECDHE-ECDSA-AES128-SHA1
+* ECDHE-ECDSA-AES256-SHA1
+* ECDHE-RSA-AES128-SHA1
+* ECDHE-RSA-AES256-SHA1
+
+
+== Upgrade your VPC via Anypoint Platform
+
+. From *Anypoint Platform*, select *Runtime Manager* > *Upgrade*.
++
+Alternatively, select *Runtime Manager* > *What's New* to read more about the VPC upgrade, and click *Continue to Upgrade* to start upgrading your VPCs.
+. Click *Upgrade Now*, under the *Upgrade Status* column for the VPC you want to upgrade.
++
+The *Upgrade to CloudHub 2.0* review page displays the configurations currently available in your VPC, as well as the configurations that are created during the upgrade.
++
+image::upgrade-ch2-review-conf.png[]
++
+. Click *Next* to continue.
+. Provide a name for the new private space.
++ 
+Optionally, you can specify xref:ps-gather-setup-info.adoc#cidr-block[reserved CIDRs] to connect to your new private space.
+. Click *Create* to create the new private space.
++
+The *Upgrade to CloudHub 2.0* review page shows the progress of the VPC upgrade process, which takes approximately fifteen to thirty minutes.
+
+As soon as the upgrade process starts, the upgraded VPC is locked for any further updates. 
+
+You can cancel the upgrade process to revert to your VPC at any time by clicking *Cancel Upgrade*. In that case, the newly created private space is deleted, and your VPC is unlocked.
+
+Once you have successfully provisioned the private space, the *Cancel Upgrade* button is no longer visible. However, you can still roll back the upgrade by deleting the private space.
+
+
+== Next Steps after VPC Upgrade
+
+After you have completed the upgrade, the private space is fully provisioned and you can start deploying applications into it.
+
+You can start migrating your applications individually by creating new versions in the CloudHub 2.0 private space and then deleting the corresponding application in the CloudHub VPC. Once you’ve successfully migrated all your applications, make sure to delete your old VPC and any attached Dedicated Load Balancers on CloudHub.
+
+During your migration, you can continue to deploy and manage applications in your VPC to maintain business continuity. However, after the upgrade, your CloudHub Dedicated VPC is locked for any infrastructure changes to prevent configuration drift. This includes adjustments to environment or business group associations, network connections, and internal DNS. Make all changes in the newly created private space instead. Any configuration changes carried over to the CloudHub 2.0 private space are applied to the CloudHub VPC during your upgrade. 
```

Comment:
```suggestion
During your migration, you can continue to deploy and manage applications in your VPC to maintain business continuity. After the upgrade, your CloudHub dedicated VPC is locked for any changes to the infrastructure, including environment or business group associations, network connections, and internal DNS, to prevent configuration drift. Make all changes in the newly created private space instead. Any configuration changes carried over to the CloudHub 2.0 private space are applied to the CloudHub VPC during your upgrade. 
```